var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -186,7 +188,11 @@ class Acts(CMakePackage, CudaPackage):
     variant(
         "benchmarks", default=False, description="Build the performance benchmarks", when="@0.16:"
     )
-    _cxxstd_values = (conditional("14", when="@:0.8.1"), "17", conditional("20", when="@24:"))
+    _cxxstd_values = (
+        conditional("14", when="@:0.8.1"),
+        conditional("17", when="@:35"),
+        conditional("20", when="@24:"),
+    )
     variant(
         "cxxstd",
         default="17",
```

Comment:
Does this still work?? `_cxxstd_values` now doesn't have 17, but the default is set to 17. What does it pick as default in practice? I though this is one of those cases where we have to duplicate the variant directive with different when clauses to have the default change accordingly.